source/whats-new.txt

```diff
@@ -27,6 +27,20 @@ Learn what's new in:
 * :ref:`Version 3.10 <version-3.10>`
 * :ref:`Version 3.9 <version-3.9>`
 
+.. _version-4.1:
+
+What's New in 4.1
+-----------------
+
+The v4.1 driver release includes the following new features:
+
+- Adds support for BSON Binary Vector. 
+  
+  For more information, see the
+  :manual:`BSON Types </reference/bson-types/>` manual page, and a usage example
+  in the :github:`mongo-cxx-driver repository </mongodb/mongo-cxx-driver/blob/master/examples/bsoncxx/bson_binary_vector.cpp>`
+  on GitHub.
```

Comment:
S: you can make this into a list for readability. I'd also link to the "Binary Data" section of that manual page:

```suggestion
  For more information, see the following resources:
  
    - :manual:`Binary Data BSON Types </reference/bson-types/#binary-data>` in the {+mdb-server+} manual
    - :github:`BSON Binary Vector </mongodb/mongo-cxx-driver/blob/master/examples/bsoncxx/bson_binary_vector.cpp>`
  usage example on GitHub.
```